uploader.json

```diff
@@ -1775,6 +1782,14 @@
             ],
             "symbolic_name": "vdsm_log"
         },
+	{
+            "file": "/etc/vdsm/logger.conf",
+            "pattern": [
+		'loggers',
+		'formatters'
```

Comment:
I cannot see any filters for Specs.vdsm_conf in the Parser in your [PR-1129](https://github.com/RedHatInsights/insights-core/pull/1129), this Ini Configure file shouldn't be filtered. And any unreasonable filter will cause parse Error in the IniConfigFile.

Reply:
Reverting pattern for `vdsm_logger_conf` as empty list